test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightWriterTests.cs

```diff
@@ -1311,6 +1312,95 @@ public async Task WriteBinaryValueToStream_WithODataUtf8JsonWriter_Async()
                 result);
         }
 
+        [Fact]
+        public async Task WriteStringValueToTextWriter_WithODataUtf8JsonWriter_Async()

```

Comment:
I would suggest adding more tests. Specifically tests which larger byte array and text values, where different "Write" calls may have different lengths, etc. Tests that will ensure the chunking and automatic flushing is tested, tests that ensure that there are left-over bytes from a single write call, etc. Maybe we should add those tests in the `JsonWriteBaseTests` class to ensure both `ODataUtf8JsonWriter` and `JsonWriter` are tested.